### DISCUSSION TITLE
min(initializer_list<_Tp> __l, _Compare __comp) Error while compiling ESP32 library code from Arduino IDE

### DISCUSSION THREAD

**Discussion by GunashekarG** [jrowberg/i2cdevlib#605] [Q&A]
OS: Linux(Ubuntu Mate)
IDE: Arduino IDE
Example: asm_terminal (USB library code for interfacing ARDUINO USB HOST SHIELD with ESP WROOM 32)

I am working to interface Arduino USB host shield with ESP WROOM 32. I have added the ESP 32 board to the Arduino IDE and able to run basic example codes on ESP32 by burning the ino files from Arduino IDE. 

Now,  I am using this link: https://github.com/ebrahim-Qaashah/USB_Host_Shield_2.0_Support_ESP32 which has the ESP32 support for the USB Host shield. The main library code for Arduino UNO is here: https://github.com/felis/USB_Host_Shield_2.0

Now, when I am compiling one of the example code asm_terminal.ino, I am getting the following error while compiling.

![ESP32_Arduino_Error](https://user-images.githubusercontent.com/31695652/102454064-637cdf00-4063-11eb-9ec8-98a972f52bad.png)

Can anyone please help me in resolving this issue. Thanks in advance.

**Comment by jrowberg** [comment]
This sounds like an issue with the USB Host Shield code, separate from I2Cdevlib. If you’re having trouble compiling one of their provided examples, have you tried asking that dev team for support?
